libs/ledger-live-common/src/exchange/swap/hooks/useSwapTransaction.ts

```diff
@@ -1,4 +1,4 @@
-import { AmountRequired } from "@ledgerhq/errors";
+import { AmountRequired, NotEnoughGas, NotEnoughGasSwap } from "@ledgerhq/errors";
```

Comment:
Could you explain to me the difference between "NotEnoughGas" and "NotEnoughGasSwap"?

Reply:
Sure, NotEnoughGas is the original error message. This is used in other parts of LLD and LLM.  NotEnoughGasSwap is the new error message with which will only be displayed on the SwapPage

Reply:
I wonder if this error is not swap specific...

Reply:
It would be good add a comment as it is a little confused.
"Specific error for Payment required with parent currency has been implemented only for swap, not in other flows as receive"